api/resolvers/notification_emails.py

```diff
@@ -0,0 +1,95 @@
+import os
+
+from itsdangerous import URLSafeTimedSerializer
+from notifications_python_client import NotificationsAPIClient
+
+from functions.email_templates import (
+    email_verification_template,
+    password_reset_template)
+from schemas.notification_email import (
+    Content,
+    Template,
+    NotificationEmail
+)
+
+NOTIFICATION_API_KEY = os.getenv('NOTIFICATION_API_KEY')
+NOTIFICATION_API_URL = os.getenv('NOTIFICATION_API_URL')
+
+SUPER_SECRET_KEY = os.getenv('SUPER_SECRET_KEY')
+SUPER_SECRET_SALT = os.getenv('SUPER_SECRET_SALT')
+
+notifications_client = NotificationsAPIClient(
+    NOTIFICATION_API_KEY,
+    NOTIFICATION_API_URL,
+)
+
+
+def resolve_send_password_reset(self, info, email):
+    template_id = password_reset_template()
+    password_reset_serial = URLSafeTimedSerializer(SUPER_SECRET_KEY)
+
+    # TODO: Change to deployment URL
+    password_reset_url = "http://localhost:3000/reset-password/" + password_reset_serial.dumps(
```

Comment:
You should be able to get the url through flask requests. [Docs](https://flask.palletsprojects.com/en/1.1.x/api/#incoming-request-data)